paddle/fluid/eager/auto_code_generator/final_state_generator/eager_gen.py

```diff
@@ -1620,7 +1652,21 @@ def GenerateNodeDefinition(self, next_grad_node_creation_str,
 
         grad_api_args_str = ", ".join(grad_api_args)
 
-        grad_function_call_str = f"""
+        if is_invoke_forward_api:
```

Comment:
Maybe we should provide fast through path. We should call phi api if we don't need next level backward, and call dygraph_function if we need it. Otherwise we will make these api slow....

Reply:
done